docker/templates/devel.template

```diff
@@ -0,0 +1,42 @@
+RUN apt-get update \
+ && DEBIAN_FRONTEND=noninteractive apt-get install -y \
+    libgflags-dev \
+    libgoogle-glog-dev \
+    libopencv-dev \
+    libleveldb-dev \
+    libsnappy-dev \
+    liblmdb-dev \
+    libhdf5-serial-dev \
+    libprotobuf-dev \
+    protobuf-compiler \
+    libatlas-base-dev \
+    python-dev \
+    python-pip \
+    python-numpy \
+    python-scipy \
+    gfortran
+
+# Use --no-install-recommends for the boost libs.
+RUN apt-get install -y --no-install-recommends \
+    libboost-all-dev
+
+# Install utilities used for the build.
+RUN apt-get install -y \
+    build-essential \
+    cmake \
+    cmake-curses-gui \
+    git \
+    ca-certificates \
+    bc \
+    wget
+
+# Install the python requirements.
+RUN wget https://raw.githubusercontent.com/BVLC/caffe/master/python/requirements.txt -O /tmp/requirements.txt && \
```

Comment:
I've never used docker so I'm definitely not an expert, but is it necessary to wget requirements.txt like this rather than getting it from the (presumably already downloaded) local copy of caffe?